### DISCUSSION TITLE
Customize footer padding

### DISCUSSION THREAD

**Discussion by vedxp** [vuejs/vitepress#1308] [Q&A]
I've tried all the way I could to figure out how to customize the footer padding, but couldn't succeed.

Will be glad if someone guides me.

##### I'm trying to reduce that (reference image) too large padding (big white space) between the `Home Content` and `Footer Section`

<details><summary>Reference Image</summary>

![chrome_screenshot_1662623941764](https://user-images.githubusercontent.com/87353286/189068734-8e25cbc6-6ead-4722-b20b-136570dbf9bf.png)

</details>

**Comment by brc-dd** [comment]
Show us your code.

**Reply by vedxp** [reply]
Hello sir,
I meant to say that I've tried to find in the documentation about changing footer padding size, but there's nothing mentioned about it.

And, I don't know either js nor vue.

So, I'm seeking for help to reduce that big [white space](https://user-images.githubusercontent.com/87353286/189068734-8e25cbc6-6ead-4722-b20b-136570dbf9bf.png) between the Home Content and the Footer Section.

**I've scaffolded my site from Vitepress's default theme, and tweaked according to my needs.**

**Comment by vedxp** [comment]
@brc-dd 
Please have a look sir!
Thanks in advance.
